### ISSUE TITLE
Icons on Palemoon 25.7.0

### ISSUE THREAD

**Issue by MarkG509** [vrillusions/permit-cookies#27]
Permit Cookies 2 updated today on Palemoon 25.7.0 on Linux x64.  The "C" icon changed to a bunch of 10 icons in 2 rows.  This update works OK in Firefox 40.x.  I am attempting to attach a screenshot of the problem, where I placed the permit-cookies icon on the right-side of the URL bar:

![screenshot-startpage search engine - pale moon](https://cloud.githubusercontent.com/assets/14034950/9564891/b80b3878-4e83-11e5-8e67-6bce0104e89b.png)

That bunch of icons on the right-end of the URL bar, starting from the white down-arrow to the yellow star icon are all due to permit-cookies.

**Comment by vrillusions** [comment]
Interesting, none of those are included in the add-on so not sure where they are coming from.  I'll bring up a linux vm and see if I can figure it out.

**Comment by MarkG509** [comment]
Thanks, please let me know if you need any more info, etc.. I'm on Linux Mint 17.2 (based on Ubuntu 14.04 LTS).  One more detail is that when I select Customize, that whole group of icons moves together, as though it were one large clump.

In the mean time, I've gone back to the previous version of Permit Cookies in Palemoon, though as I've said, the latest Permit Cookies 2 works fine on Firefox 40.0.3 on the same Linux box.

**Comment by vrillusions** [comment]
Good news is I'm also experiencing that issue on Xubuntu 14.04 but I'm not seeing any errors in browser console.  Thinking out loud at a couple reasons for this:

* Before it checked to make sure the `addonBar` variable had a value before working on it and now it doesn't [see here](https://github.com/vrillusions/permit-cookies/commit/0f07d193fa5897bc7f6d4bbc7f0df258c1f5f156#diff-7bfd5feef9b59cc1434f3f8b595737bd)
* If I have the toolbar show both text and icons I see the text for permit cookies there. That could mean the way I dynamically load the icon set may not be supported in Pale moon

The solution to this may just be uploading v1.0.1 to palemoon add on site since I doubt there's a lot of users of it (this is the first time I've heard of pale moon).  Nothing I added in v2.0 is critical and I imagine the api change that's happening in Firefox 42 isn't going to apply to pale moon.  I did update the description on the add-on page that pale moon users should stick with v1.0.1
